TrecQA/build_qrels.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+for set in ['train-all', 'raw-test', 'raw-dev']:        
+    
+    with open(set + '.qrel', 'w') as qrelf:
+        labels = [y.strip() for y in open(os.path.join(set+'/sim.txt')).readlines()]
+        qids = [id.strip() for id in open(os.path.join(set+'/id.txt')).readlines()]
+
+        for i in range(len(labels)):
+            qid = qids[i]
+            lbl = labels[i]
+            ext_feats = None
```

Comment:
Fixed. zipping here may not be in convention with code elsewhere in the project that reads in labels. But good to know for the future.